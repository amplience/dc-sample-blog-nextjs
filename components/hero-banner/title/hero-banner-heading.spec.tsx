import React from 'react';
import renderer from 'react-test-renderer';
import HeroBannerHeading from './hero-banner-heading';

describe('HeroBannerHeading', () => {
  test('renders hero banner title', async () => {
    const component = await renderer.create(<HeroBannerHeading heading='Test title' />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
