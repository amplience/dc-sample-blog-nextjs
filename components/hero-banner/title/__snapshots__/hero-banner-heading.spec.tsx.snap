// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroBannerHeading renders hero banner title 1`] = `
<h1
  className="jsx-641291359"
>
  Test title
</h1>
`;
